app/Events/PostTagged.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Rogue\Events;
+
+use Rogue\Models\Tag;
+use Rogue\Models\Post;
+use Illuminate\Broadcasting\Channel;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Broadcasting\PrivateChannel;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Broadcasting\InteractsWithSockets;
+
+class PostTagged
+{
+    use Dispatchable, InteractsWithSockets, SerializesModels;
```

Comment:
Since this event is internal to the application (it's just triggering a listener class), I think we can kill `InteractsWithSockets` and the `broadcastOn` methods here. 📻